src/Pim/Bundle/DataGridBundle/Normalizer/ProductNormalizer.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Pim\Bundle\DataGridBundle\Normalizer;
+
+use Pim\Bundle\CatalogBundle\Filter\CollectionFilterInterface;
+use Pim\Component\Catalog\Model\ProductInterface;
+use Pim\Component\Catalog\Model\ProductValueCollectionInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\SerializerAwareNormalizer;
+
+/**
+ * Product normalizer for datagrid
+ *
+ * @author    Marie Bochu <marie.bochu@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ProductNormalizer extends SerializerAwareNormalizer implements NormalizerInterface
+{
+    /** @var CollectionFilterInterface */
+    private $filter;
+
+    /**
+     * @param CollectionFilterInterface $filter The collection filter
+     */
+    public function __construct(CollectionFilterInterface $filter)
+    {
+        $this->filter = $filter;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function normalize($product, $format = null, array $context = [])
+    {
+        if (!$this->serializer instanceof NormalizerInterface) {
+            throw new \LogicException('Serializer must be a normalizer');
+        }
+
+        $context = array_merge(['filter_types' => ['pim.transform.product_value.structured']], $context);
+        $data = [];
+        $locale = current($context['locales']);
+
+        $data['identifier'] = $product->getIdentifier();
+        $data['family'] = $this->getFamilyLabel($product, $locale);
+        $data['groups'] = $this->getGroupsLabels($product, $locale);
+        $data['enabled'] = (bool) $product->isEnabled();
```

Comment:
the type casting shouldnt be done here, it should be the job of the function isEnabled() to always returns a boolean.